### ISSUE TITLE
GeoJSON is missing from production build

### ISSUE THREAD

**Issue by carpeliam** [codeforboston/cambridge_voting_locations#60]
Steps to reproduce:
1. Create production build (`npm run build`)
2. Open website (e.g., `cd dist && python -m SimpleHTTPServer`)
3. Perform some kind of action that uses GeoJSON, for example, find an election day polling location
4. Verify that a JavaScript error occurs

I've narrowed it down to GeoJSON by modifying the uglify config in Grunt:

``` javascript
grunt.initConfig({
  uglify: {
    options: {
      mangle: false,
      compress: false
    }
  }
  /* ... */
});
```

**Comment by carpeliam** [comment]
@moczka sounds awesome! I'm going to assign #40 to you then - feel free to unassign it if you put it back down.

**Comment by moczka** [comment]
@carpeliam Great thanks. I wouldn't mind if someone could tag along to fix #40 with me in case I can't find anything. I will try my best, keep everyone up to date on my findings.

**Comment by moczka** [comment]
GeoJson is no longer a dependency. Closing issue
